packages/gestalt-design-tokens/config.json

```diff
@@ -98,6 +98,7 @@
     },
     "android": {
       "transformGroup": "android-custom",
+      "options": { "fileHeader": "androidCustomHeader" },
```

Comment:
what's this?

Reply:
Update to the file headers for android to remove the timestamp